Project 1/static/script.js

```diff
@@ -0,0 +1,97 @@
+function book_html(id) {
```

Comment:
@lakshmikiran14 Where are you using these functions defined in script.js?

Reply:
@lakshmikiran14 I want to know in which html file you are using this script.js?

Reply:
My task was to implement this page and this function is getting called in search api. Whener the book is selected in the search page, this function will be called

Reply:
As it is a single page app, i have to link it in search. I'll do it @TulasiJagan49

Reply:
@lakshmikiran14 Okie